test/sider/oppfolgingsplaner/OppfolgingsplanerOversiktTest.tsx

```diff
@@ -102,6 +106,10 @@ describe("OppfolgingsplanerOversikt", () => {
   });
 
   describe("Be om oppfølgingsplan", () => {
+    beforeEach(() => {
+      queryClient.clear();
```

Comment:
Kunne man unngått denne hvis man tok `beforeEach` på rotnivå i stedet for under hver describe i denne filen?